vhdbuilder/packer/windows/components_json_helpers.tests.ps1

```diff
@@ -353,6 +353,12 @@ Describe 'Tests of components.json' {
         $components | Should -Contain "mcr.microsoft.com/oss/kubernetes/pause:3.9"
     }
 
+    it 'has the right version of ciprod' {
+        $components = GetComponentsFromComponentsJson $componentsJson
+
+        $components | Should -Contain "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.24"

```

Comment:
Just an FYI. I added a test_components.json for my unit test to avoid coming back to change some values that could be updated in the future. For example, in this case, `ciprod:win-3.1.24`.

Reply:
Yup - I contemplated that, but then wanted to have comprehensive tests for windows at first to make sure I had the right versions in the file. Will likely remove these over time. I also use a sample components.json for other tests.